src/wp-admin/includes/menu.php

```diff
@@ -368,7 +368,17 @@ function sort_menu( $a, $b ) {
 }
 unset( $last_menu_key );
 
-if ( ! user_can_access_admin_page() ) {
+if ( ! admin_page_exists() ) {
+
+	/**
+	 * Fires when access to an admin page which does not exist.
+	 *
+	 * @since 6.9.0
```

Comment:
```suggestion
	 * @since 7.0
```
Should we change @since 6.9.0 to @since 7.0?

Reply:
```suggestion
	 * @since 7.0.0
```